istioctl/cmd/kubeinject.go

```diff
@@ -394,6 +384,17 @@ func setupKubeInjectParameters(sidecarTemplate *inject.Templates, valuesConfig *
 				return nil, nil, err
 			}
 		}
+		return injector, nil, nil
```

Comment:
basically if we get the internal injector, return immediately. Any other config is not relevant so no need to try to read it.